automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/SamApiSpec.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.firecloud.test.api
```

Comment:
We're intending this to move to the Sam repo eventually, right? I wonder if would make sense to name the package:
```
org.broadinstitute.dsde.firecloud.test.api.sam
```
so it's clear this is a Sam test. Similarly for Leonardo, Rawls, etc API test. It might make refactoring later easier.